auction9-backend/serverless.yml

```diff
@@ -27,4 +27,13 @@ provider:
     MYSQL_PORT: 8889
     MYSQL_DATABASE: auction9
     MYSQL_USER: root
-    MYSQL_PASSWORD: root
\ No newline at end of file
+    MYSQL_PASSWORD: root
+
+functions:
+  # return 'auction' by id
+  getAuction:
+    handler: handler.getAuction
+    events:
+      - http:
+          path: auction/{id}
```

Comment:
Resource names in REST APIs should be in plural, so `auctions/{id}`

Reply:
Change applied